README.md

```diff
@@ -1 +1,139 @@
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Project-OOP-School-Library <a name="about-project"></a>
+
+The Project OOP-school-library is an console app interacting with users, where you can store the record of what books are in the library and who borrowed them and their detials. So, in this project you can add new books, new students, new teachers. And you can also view the details regarding the books, students and teachers.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+<summary>Skills</summary>
+  <ul>
+    <li><a href="https://rubygems.org">Ruby</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Uses rubocop linter to improve the code structure**
+- **Can add new records of students, teachers, books**
+- **Can update the record of who borrowed a particular book basing the ID.**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+Before running this project, make sure you have the following prerequisites installed:
+
+<ul>
+  <li>Ruby</li>
+  <li>rubocop</li>
+</ul>
+
+### Setup
+
+1. Clone this repository to your desired folder:
+
+   ```sh
+   git clone https://github.com/the-faizmohammad/School-Library-OOPs
+   ```
+
+2. Navigate to the project directory:
+
+    ```sh
+    cd directory name
+    ```
+
+### Usage
+
+To run the project:
+
+```sh
+  ruby person.rb
+```
+
+### Run tests
+
+To run tests after setting up the linters, run the following command:
+
+```sh
+  rubocop
+```
+## 👥 Author <a name="authors"></a>
+
+👤 **Faiz Mohammad**
+
+- GitHub: [@githubhandle](https://github.com/HossainAraf)
+- Twitter: [@gmail](email2faizm@gmail.com)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/faiz-mohammad-967354142/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+As this project progresses, I plan to implement the following features:
+
+- **Add the basic UI**
+
+Stay tuned for updates and enhancements to the vet clinic database!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for providing the required lessons and tutorials for successfully completing the project.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Your license file has no content, kindly make  sure you add the license content to make your project look more professional